pyproject.toml

```diff
@@ -0,0 +1,78 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lsst-meas-extensions-multiprofit"
+authors = [
+    {name="Rubin Observatory Data Management", email="dm-admin@lists.lsst.org"},
+]
+description = "Tasks and pipelines for running multiprofit with the LSST Butler"
+readme = "README.rst"
+requires-python = ">=3.10"
+keywords = ["astronomy", "lsst", "galaxy"]
+license = {file = "LICENSE"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+]
+dependencies = [
+    "astropy",
+    "galsim",
+    "gauss2d",
+    "gauss2dfit",
+    "lsst-multiprofit",
```

Comment:
This implies you are putting `lsst-multiprofit` on PyPI. Are you? I know this package itself can't go on PyPI because of the meas_base dependency.

Reply:
Does it imply availability on PyPI? Once multiprofit is scons-built, it shows up on pip list with the same version as it would if pip-installed. On the other hand, gauss2d(fit) don't when eups-built because I skipped the pip install step, figuring it would be unnecessary. But perhaps none of this matters because presumably this package won't be pip-installable until all of the dependencies like meas_base are too...

Reply:
`pip` will read this dependency list. It will, I think, just try to see if they are installed before trying to check on PyPI so if everything has happened beforehand if will be fine.

Yes, none of this matters because meas_base is not going to be pip installable.